modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItTablesApiTest.java

```diff
@@ -204,12 +425,135 @@ public void testCreateDropTable() throws Exception {
-                                SchemaBuilders.column("key", ColumnType.INT64).asNonNull().build(),
-                                SchemaBuilders.column("valInt", ColumnType.INT32).asNullable().build(),
-                                SchemaBuilders.column("valStr", ColumnType.string()).withDefaultValueExpression("default").build()
+                schemaName + "." + shortTableName,
+                tblCh -> convert(SchemaBuilders.tableBuilder(schemaName, shortTableName).columns(
+                        SchemaBuilders.column("key", ColumnType.INT64).asNonNull().build(),
+                        SchemaBuilders.column("valInt", ColumnType.INT32).asNullable().build(),
+                        SchemaBuilders.column("valStr", ColumnType.string())
+                                .withDefaultValueExpression("default").build()
+                        ).withPrimaryKey("key").build(),
+                        tblCh).changeReplicas(2).changePartitions(10)
+        );
+    }
+    
+    /**
+     * Adds an index if it does not exist.
+     *
+     * @param node           Cluster node.
+     * @param schemaName     Schema name.
+     * @param shortTableName Table name.
+     */
+    protected Table createTableIfNotExists(Ignite node, String schemaName, String shortTableName) {
+        return node.tables().createTableIfNotExists(
+                schemaName + "." + shortTableName,
+                tblCh -> convert(SchemaBuilders.tableBuilder(schemaName, shortTableName).columns(
+                        SchemaBuilders.column("key", ColumnType.INT64).asNonNull().build(),
+                        SchemaBuilders.column("valInt", ColumnType.INT32).asNullable().build(),
+                        SchemaBuilders.column("valStr", ColumnType.string())
+                                .withDefaultValueExpression("default").build()
                         ).withPrimaryKey("key").build(),
                         tblCh).changeReplicas(2).changePartitions(10)
         );
     }
+    
+    /**
+     * Adds an index.
+     *
+     * @param node           Cluster node.
+     * @param schemaName     Schema name.
+     * @param shortTableName Table name.
+     */
+    protected void addColumn(Ignite node, String schemaName, String shortTableName) {
+        ColumnDefinition col = SchemaBuilders.column("valStrNew", ColumnType.string()).asNullable()
+                .withDefaultValueExpression("default").build();
+    
+        addColumnInternal(node, schemaName, shortTableName, col);
+    }
+    
+    /**
+     * Adds a column according to the column definition.
+     *
+     * @param node Ignite node.
+     * @param schemaName Schema name.
+     * @param shortTableName Table name.
+     * @param colDefinition Column defenition.
+     */
+    private void addColumnInternal(Ignite node, String schemaName, String shortTableName, ColumnDefinition colDefinition) {
+        node.tables().alterTable(
+                schemaName + "." + shortTableName,
```

Comment:
can we make utility method instead numerous : schemaName + "." + shortTableName ?

Reply:
I think we must do it. But it is not a part of the issue.